.gitignore

```diff
@@ -1,6 +1,168 @@
-.idea
-.vscode
+# Source: https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Python.gitignore
+
+# Visuals
+visuals/*.html
```

Comment:
Add the `html` files generated from the visual scripts to the `.gitignore` file since they can be easily generated by running their respective scripts.